docs/data-type-support.rst

```diff
@@ -0,0 +1,6 @@
+Data type support
+******************************************
+
+hipRAND's data type support is matching cuRAND's data type support, for further information check out cuRAND's documentation. On AMD hardware the backend is provided by rocRAND. The detailed description can be found at :doc:`rocRAND's data type support page.<rocrand:data-type-support>`
```

Comment:
Recommended

hipRAND's data type support matches cuRAND's data type support.  rocRAND provides the backend on AMD hardware. 
Refer to :doc:`rocRAND's data type support page.<rocrand:data-type-support>`for more information.

Reply:
Yes, this was more or less the prior statement, but as a developer it would be slightly confusing to me, that we state, that the support is matching cuRAND and immediately after that refer to rocRAND, which is supposed to be a superset of cuRANDs (and hipRANDs).

If we don't want to refer to cuRAND or nvidia documentation, than this is fine with me.

Reply:
Thanks for the changes. Users will refer to CUDA if needed. We should consider providing the information on our portal for context without sending users to CUDA documentation.

Reply:
Okay, that makes sense to me. We actually have a comparison to the cuRANDs support on rocRANDs data type support page, so I added a line stating that.

Reply:
That's awesome. Appreciate the changes @MKKnorr. The intent is to get our users to use our documentation unless, contextually, they are required to access a third-party website.